### PR TITLE
Feature/cpa 204

### PR DESCRIPTION
**Pull request by pmercado23**
# 

Creating filters on swagger page for devs

## 🗣 Description

adding in filters onto swagger page for serialization and basic queries. 

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: controller/src/api/serializers/customer_serializers.py [hunk 1]
```diff
@@ -167,3 +171,26 @@ class SectorGetSerializer(serializers.Serializer):
     name = serializers.CharField(max_length=250)
     industries = SectorIndustry(many=True)
 
+
+class CustomerQurySerializer(serializers.Serializer):
```
Comment (zenine07) [inline controller/src/api/serializers/customer_serializers.py]: Typo - Should be CustomerQuerySerializer

**Review by zenine07 — COMMENTED** [review]
Everything looked good except the typo

File: controller/src/api/serializers/customer_serializers.py (hunk 1 above)
Comment (pmercado23) [reply controller/src/api/serializers/customer_serializers.py]: ah, good catch, updated now

**Review by zenine07 — APPROVED** [review]
LGTM

**Pull request merged**